.expeditor/integration.resources.yml

```diff
@@ -1 +1,20 @@
-#
+---
+expeditor:
+  defaults:
+    buildkite:
+      timeout_in_minutes: 45
+      retry:
+        automatic:
+          limit: 1
+
+steps:
+  - label: integration-test-kitchen
+    command:
+      - RAKE_TASK=test:integration /workdir/.expeditor/buildkite/verify.sh
+    expeditor:
+      executor:
+        docker:
+          environment:
+            - CONCURRENCY: 5
+            - DOCKER: 1
```

Comment:
I think we need to set `KITCHEN_YAML=./kitchen.dokken.yml` here to avoid kitchen trying to run the default `kitchen.yml`